src/Sticky.jsx

```diff
@@ -127,10 +127,11 @@ class Sticky extends Component {
 
     /**
      * Update the initial position, width, and height. It should update whenever children change.
-     * @param {Object} options optional top and bottomBoundary new values
+     * @param {Object} options optional top and bottomBoundary new values.
+     * Set updateAfterInitialDimension to false to disable update after dimensions retrieval
      */
     updateInitialDimension (options) {
-        options = options || {}
+        options = options || { updateAfterInitialDimension: true }
```

Comment:
How does a user actually change this option, I didn't see a way to pass in the value.

If they can update it, what happens when `false` is passed in? How does it get updated?

```suggestion
        options = options || { updateAfterInitialDimension: true };
```